src/i18n/strings/en_EN.json

```diff
@@ -3782,6 +3782,7 @@
         "ignore_confirm_title": "Ignore %(user)s",
         "invited_by": "Invited by %(sender)s",
         "jump_to_rr_button": "Jump to read receipt",
+        "jump_to_rr_button_disabled": "This feature is disabled",
```

Comment:
This feels a bit overly unhelpful. Conditional that shows either, "Not available in spaces" or, "No receipts for ${user}" depending on isSpace?

Reply:
The tooltip is gone, but this would have been as specified by design. See https://github.com/element-hq/element-web/issues/27883#issuecomment-2268816781